tools/packaging/kata-deploy/Dockerfile

```diff
@@ -3,27 +3,28 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Specify alternative base image, e.g. clefos for s390x
-ARG IMAGE
-FROM ${IMAGE:-registry.centos.org/centos}:7
```

Comment:
This change doesn't belong to cd2aaeda2a07d2605634cc381b1bb3d68fe562a6 AFAICT. You fixed a conflict manually, right ?

Cherry picking bd1ed26c8d0ef8da99bc2ffffb099f93f73a3880 and 6267909501a1dc221e94ec2c6e80c7c0e1c083e7 seems to be enough to avoid that.

Reply:
Exactly, but I really would like to avoid cherry-picking GitHub actions that should never ever be part of this branch into here.